### DISCUSSION TITLE
My first CRUD App Developed with SQLPage

### DISCUSSION THREAD

**Discussion by a-gonzalo** [sqlpage/SQLPage#241] [Show and tell]
Hi everyone,
Today I finalized the first version of "Better Fiori", a webapp to report and control my work hours. It's probably not useful to anyone, but I'm proud of it.

I made it because I need a more specific control on my reports that asking my boss to them, so I built it. Now I can do some statistics and querys into my reports, thanks to SQLPage!!

This is the GitHub [repo](https://github.com/Ajotah98/BetterFiori).

![image](https://github.com/lovasoa/SQLpage/assets/91145535/11a1b4e8-1ddc-4297-827d-d0b8800da0d6)
![image](https://github.com/lovasoa/SQLpage/assets/91145535/4c2d40ef-1048-485e-a714-1ddd90f94d85)
![image](https://github.com/lovasoa/SQLpage/assets/91145535/7febf2b1-0b2f-441e-8a4b-aa95e1614a56)

Probably is not useful, but I also encourage all of you to go there if you are stucked doing something. It could help <3.

**Comment by lovasoa** [comment]
Your app really looks great ! Is it open source, can one see the source code somewhere ?

Just out of curiosity, how long did you spend on it, and how long do you think it would have taken you with a traditional tech stack ?

**Reply by a-gonzalo** [reply]
Hi! Yes, it's open-source, so you can use it as an example if you want 😊. The link is in the post.

The time I spent is approximately between 4 and 5 hours, considering I need to read the docs for almost everything.

If it had depended on doing it with a standard framework like React, or something like that, I wouldn't have even finished it. I've programmed things with JavaFX, internal applications, and such, and it hasn't been especially difficult for me. The problem is that I'm terrible at design, and it has even caused me anxiety to have to learn CSS. So, if I had had to make it a web application, I wouldn't have even started it.

Even so, if I had made it as a Java application, the total development time would probably have been around 8-10 hours minimum.

**Reply by lovasoa** [reply]
Sorry I had missed the link !

I'll quote you on that, I think it's interesting feedback that may convince more people to try sqlpage !

**Comment by matthewlarkin** [comment]
Yes!! 🙌 🙌

Great job @Ajotah98!
